core-web/libs/ui/src/lib/directives/dot-select-item/dot-select-item.directive.ts

```diff
@@ -1,28 +0,0 @@
-import { Directive, HostListener, Optional, Self } from '@angular/core';
-
-import { AutoComplete } from 'primeng/autocomplete';
-
-@Directive({
-    selector: 'p-autoComplete[dotSelectItem]',
-    standalone: true
-})
-export class DotSelectItemDirective {
```

Comment:
This directive is used in other parts, are we sure to delete it?

Reply:
Yes, I took a look but I can double-check.